.github/workflows/updatenet-delete.yml

```diff
@@ -39,7 +39,8 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_DEVNET_SECRET_ACCESS_KEY }}
           ecr-public-registry: ${{ vars.ECR_PUBLIC_REGISTRY }}
           argo-sync-user-token: ${{ secrets.ARGO_SYNC_USER_TOKEN }}
-          repo-apps-name: ${{ secrets.REPO_OPS_FEATURENETS_NAME }}
+          repo-apps-name: ${{ secrets.REPO_ARGOCD_APPS_NAME }}
+          repo-featurenets-name: ${{ secrets.REPO_OPS_FEATURENETS_NAME }}
```

Comment:
Why this is called `REPO_OPS_FEATURENETS_NAME` and not just `REPO_FEATURENETS_NAME`? Since that repo does not contain ops.sh file, repo `REPO_ARGOCD_APPS_NAME` does